rackcity/admin.py

```diff
@@ -1,7 +1,8 @@
 from django.contrib import admin
 
-from .models import Asset, ITModel, Rack
+from .models import Asset, Log, ITModel, Rack
 
 admin.site.register(Asset)
+admin.site.register(Log)
```

Comment:
What is this and should I have added models that I created here 😳

Reply:
it just lets you look at the table from django `/admin/` endpoint, just wanted it while testing not super impactful

Reply:
That looks helpful actually I've never used it